intermittent_tracker/flask_server.py

```diff
@@ -54,6 +54,11 @@ def dashboard_query():
 def index():
     return send_from_directory('static', 'index.html')
 
+@app.route('/flaky_tests')
+def flaky_tests():
```

Comment:
It's great that you made the method in `dashboard.py` take a parameter for number of tests, but can you also allow passing a URL parameter here to control that as well?